### ISSUE TITLE
kojiapi: ability to cancel compose

### ISSUE THREAD

**Issue by gicmo** [osbuild/osbuild-composer#994]
Currently there is no API to cancel a compose. But there is the ability to cancel a task in koji, which should then cancel the compose.

**Comment by teg** [comment]
Makes sense. Under the hood we support cancellation, just a matter of hooking it up.
